src/daemon/Display.cpp

```diff
@@ -365,27 +372,33 @@ namespace SDDM {
         return false;
     }
 
-    void Display::startAuth(const QString &user, const QString &password, const Session &session) {
+    bool Display::startAuth(const QString &user, const QString &password, const Session &session) {
 
         if (m_auth->isActive()) {
             qWarning() << "Existing authentication ongoing, aborting";
-            return;
+            return false;
         }
 
         m_passPhrase = password;
 
         // sanity check
         if (!session.isValid()) {
             qCritical() << "Invalid session" << session.fileName();
-            return;
+            return false;
         }
         if (session.xdgSessionType().isEmpty()) {
             qCritical() << "Failed to find XDG session type for session" << session.fileName();
-            return;
+            return false;
         }
         if (session.exec().isEmpty()) {
             qCritical() << "Failed to find command for session" << session.fileName();
-            return;
+            return false;
+        }
+        // check if the user even exists
+        struct passwd *pw = getpwnam(qPrintable(user));
+        if (!pw) {
+            qCritical() << "Failed to find the autologin user" << user;
```

Comment:
```suggestion
            qCritical() << "Failed to find the user" << user;
```

This path isn't autologin specific